src/elements/common/upload-dialog/UploadDialog.js

```diff
@@ -56,9 +58,11 @@ const UploadDialog = ({
         portalClassName={`${CLASS_MODAL} be-modal-upload`}
     >
         <ContentUploader
+            {...contentUploaderProps}
             apiHost={apiHost}
             onClose={onClose}
             onComplete={onUpload}
+            contentUploaderProps={contentUploaderProps}
```

Comment:
This line seems unnecessary. ContentUploader doesn't have a prop name contentUploaderProps

Reply:
Removed